### PR TITLE
Firefox iOS v13.0 (20180710 1440)

### PR DESCRIPTION
**Pull request by farhanpatel**

### CODE REVIEW

File: en-US/firefox-ios.xliff [hunk 1]
```diff
@@ -3083,6 +3157,18 @@
       </trans-unit>
     </body>
   </file>
+  <file original="Client/URL.strings" source-language="en" datatype="plaintext" target-language="en">
```
Comment (st3fan) [inline en-US/firefox-ios.xliff]: This should not be in its own `URL` table.

File: en-US/firefox-ios.xliff [hunk 2]
```diff
@@ -3083,6 +3157,18 @@
...
+    <header>
+      <tool tool-id="com.apple.dt.xcode" tool-name="Xcode" tool-version="8.3.3" build-num="8E3004b"/>
+    </header>
+    <body>
+      <trans-unit id="Paste &amp; Go">
```
Comment (st3fan) [inline en-US/firefox-ios.xliff]: Needs a proper ID.

File: en-US/firefox-ios.xliff [hunk 3]
```diff
@@ -3029,6 +3091,18 @@
       </trans-unit>
     </body>
   </file>
+  <file original="Client/SearchWithFirefox.strings" source-language="en" datatype="plaintext" target-language="en">
```
Comment (st3fan) [inline en-US/firefox-ios.xliff]: Should not be in its own table. Should end up in Localizable.strings instead.

File: en-US/firefox-ios.xliff [hunk 4]
```diff
@@ -3029,6 +3091,18 @@
...
+    <header>
+      <tool tool-id="com.apple.dt.xcode" tool-name="Xcode" tool-version="8.3.3" build-num="8E3004b"/>
+    </header>
+    <body>
+      <trans-unit id="Search with Firefox">
```
Comment (st3fan) [inline en-US/firefox-ios.xliff]: Needs a proper ID.

File: en-US/firefox-ios.xliff [hunk 5]
```diff
@@ -1789,11 +1806,6 @@
         <target>Open in New Tab</target>
         <note>Context menu item for opening a link in a new tab</note>
       </trans-unit>
-      <trans-unit id="Open in…">
```
Comment (st3fan) [inline en-US/firefox-ios.xliff]: Needs a proper ID

File: en-US/firefox-ios.xliff [hunk 6]
```diff
@@ -1769,6 +1781,11 @@
         <target>Oops! Firefox crashed</target>
         <note>Title for prompt displayed to user after the app crashes</note>
       </trans-unit>
+      <trans-unit id="Open App Store">
```
Comment (st3fan) [inline en-US/firefox-ios.xliff]: Needs a proper ID.

File: en-US/firefox-ios.xliff [hunk 7]
```diff
@@ -1749,6 +1756,11 @@
         <target>No tabs</target>
         <note>Message spoken by VoiceOver to indicate that there are no tabs in the Tabs Tray</note>
       </trans-unit>
+      <trans-unit id="Not Now">
```
Comment (st3fan) [inline en-US/firefox-ios.xliff]: Needs a proper ID.

File: en-US/firefox-ios.xliff [hunk 8]
```diff
@@ -1157,8 +1157,8 @@
         <note>Button label to temporarily continue to the site from the certificate error page</note>
       </trans-unit>
       <trans-unit id="ExternalLink.AppStore.ConfirmationTitle">
-        <source>Open this link in the App Store app?</source>
-        <target>Open this link in the App Store app?</target>
+        <source>Open this link in the App Store?</source>
```
Comment (st3fan) [inline en-US/firefox-ios.xliff]: @Delphine do we get in trouble with Pontoon if we make changes like this? Or does this simply invalidate the string?

File: en-US/firefox-ios.xliff [hunk 9]
```diff
@@ -1749,6 +1756,11 @@
         <target>No tabs</target>
         <note>Message spoken by VoiceOver to indicate that there are no tabs in the Tabs Tray</note>
       </trans-unit>
+      <trans-unit id="Not Now">
+        <source>Not Now</source>
+        <target>Not Now</target>
+        <note>label for Not Now button</note>
+      </trans-unit>
       <trans-unit id="Not now">
```
Comment (st3fan) [inline en-US/firefox-ios.xliff]: Better ID.

File: en-US/firefox-ios.xliff (hunk 8 above)
Comment (Delphine) [reply en-US/firefox-ios.xliff]: This should result in that the existing translation will be removed, and Pontoon will mark the existing translation as a suggestion.

**Pull request closed without merging**